sdk/src/test/java/com/mapbox/maps/MapControllerTest.kt

```diff
@@ -210,4 +215,39 @@ class MapControllerTest {
     mapController.setOnFpsChangedListener(listener)
     verify { renderer.setOnFpsChangedListener(listener) }
   }
+
+  @Test
+  fun initializePluginsEmpty() {
+    val mapController =
+      MapController(
+        renderer,
+        nativeObserver,
+        mapInitOptions,
+        nativeMap,
+        mapboxMap,
+        MapPluginRegistry(mockk()),
+        mockk()
+      )
+    val mapInitOptions = MapInitOptions(mockk(relaxed = true))
+    mapInitOptions.plugins = listOf()
+    mapController.initializePlugins(mapInitOptions)
+  }
+
+  @Test
+  fun initializePluginsCamera() {
```

Comment:
good to have a dedicated test for GesturesPlugin as well, since there are two logic branches to createPlugin.